### PR TITLE
Updating NXLog latest version to 2.10.2150

### PR DESCRIPTION
**Pull request by couilllard45682**

### CODE REVIEW

File: nxlog.sls [hunk 1]
```diff
@@ -1,5 +1,5 @@
 nxlog:
-  {% for version in ['2.10.2102',
+  {% for version in ['2.10.2150', '2.10.2102',
```
Comment (twangboy) [inline nxlog.sls]: The older versions are causing a 404 error. So, unless you can find the URL to the older installer files, we'll probably need to remove the older ones. If that's the case, there's no point in having all the jinja code. But, I'm fine if you leave it in.

**Comment by couilllard45682** [comment]
Updated to remove older and unavailable versions

File: nxlog.sls (hunk 1 above)
Comment (couilllard45682) [reply nxlog.sls]: Done

File: nxlog.sls [hunk 2]
```diff
@@ -1,7 +1,5 @@
 nxlog:
-  {% for version in ['2.10.2102',
-                     '2.9.1716'] %}
-  '{{ version }}':
+  '2.10.2150':
     full_name:  'NXLog-CE'
     installer: 'https://nxlog.co/system/files/products/files/348/nxlog-ce-{{ version }}.msi'
```
Comment (twangboy) [inline nxlog.sls]: Need to remove `{{ version }}` here and hard-code the version

Comment (twangboy) [reply nxlog.sls]: Same for the uninstaller

**Pull request merged**